appengine/standard_python3/bundled-services/mail/flask/main.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from flask import Flask, request
+from google.appengine.api import mail
+from google.appengine.api import wrap_wsgi_app
+
+app = Flask(__name__)
+app.wsgi_app = wrap_wsgi_app(app.wsgi_app, use_deferred=True)
+
+
+@app.route('/send_invalid_mail', methods=['GET'])
+def send_invalid_mail():
+    mail.send_mail(
+        sender='test-python-user@shreejad-knative-dev.appspotmail.com',
```

Comment:
Should we use a more generic `sender` value here?

Reply:
The django app does not try to send an invalid message, unlike the flask and wsgi versions. All three versions need to be made functionally as close to identical as possible. Note that the sender probably needs to be set up as a valid sending address in an allowlist, so it's hard to see how it can be hard-coded. And the invalid receiving address should be something guaranteed to bounce email.